docs/complex_parameters.md

```diff
@@ -104,7 +104,11 @@ importKey(Tezos, emailExample, passwordExample, mnemonicExample, secretExample)
 
 ## Calling the function of a contract having a complex object as a parameter
 
+<<<<<<< HEAD
```

Comment:
This seems to have been added by git (we should remove it):
```
  <<<<<<< HEAD
  =======
  >>>>>>> master
```

Reply:
yes thanks.